.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Python package
+
+on:
+  push:
+    branches:
+      - dev
+  pull_request:
+  release:
+    types: [published]
+
+
+jobs:
+  test:
+    name: Run pytest
+    # Only run on push if this is the phac-nml dev branch (merged PRs)
+    if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'phac-nml/genomic_address_service') }}"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.10", "3.11"]
```

Comment:
I wonder if you should do python versions `3.10` and `3.13` (e.g., the min and max python versions)?

Reply:
Python 3.13 causes a problem with numba

Reply:
Looks like numba cannot handle python 3.13 [Release information](https://numba.readthedocs.io/en/stable/user/installing.html)